### PR TITLE
TH-1038: Hide map from internet events.

### PR DESCRIPTION
**Pull request by melniiv**
## Additional notes :spiral_notepad:

### CODE REVIEW

File: src/domain/event/eventContent/EventContent.tsx [hunk 1]
```diff
@@ -22,6 +22,8 @@ const EventContent: React.FC<Props> = ({ event, eventType }) => {
   const locale = useLocale();
   const { description, photographerName } = getEventFields(event, locale);
 
+  const isInternetEvent = event?.location?.id === 'helsinki:internet';
```
Comment (Sartonon) [inline src/domain/event/eventContent/EventContent.tsx:25]: `'helsinki:internet'` could be stored as a constant, might be used somewhere else in the future.

**Review by Sartonon — COMMENTED** [review]
Good. Maybe store 'helsinki:internet' to a constant.

File: src/domain/event/eventContent/EventContent.tsx (hunk 1 above)
Comment (melniiv) [reply src/domain/event/eventContent/EventContent.tsx]: fixed

**Review by Sartonon — APPROVED** [review]

File: src/domain/event/eventContent/__tests__/EventContent.test.tsx [hunk 2]
```diff
@@ -88,3 +87,16 @@ it('should render event content fields', async () => {
     expect(screen.queryAllByRole(role, { name })).toHaveLength(2);
   });
 });
+
```
Comment (nenonja) [inline src/domain/event/eventContent/__tests__/EventContent.test.tsx:90]: we could also have another counterpart test that checks that map _is_ present when id is something else

**Review by nenonja — COMMENTED** [review]
One comment

**Pull request merged**